org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -22,51 +22,51 @@ <h2>Snapshot Build @qualified.bundle.version@ (@build.date@)</h2>
+      (GitHub <a href="https://github.com/jacoco/jacoco/issues/610">#610</a>).</li>
+  <li>Additional bytecode compiled for <code>synchronized</code> blocks
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/501">#501</a>).</li>
-  <li>Exclude from a report a part of bytecode that compiler generates for a
-      try-with-resources statement
+  <li>Additional bytecode compiled for try-with-resources statements
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/500">#500</a>).</li>
-  <li>During creation of a report merge duplicate instructions that compiler generates
-      for finally blocks
+  <li>Additional bytecode compiled for <code>finally</code> blocks
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/604">#604</a>).</li>
-  <li>Exclude from a report methods which are annotated with <code>@lombok.Generated</code>.
-      Initial analysis and contribution by Rüdiger zu Dohna
-      (GitHub <a href="https://github.com/jacoco/jacoco/issues/513">#513</a>).</li>
-  <li>Exclude from a report methods which are annotated with
-      <code>@groovy.transform.Generated</code> to better integrate with Groovy
-       >= 2.5.0. Thanks to Andres Almiray for adding the annotation to Groovy
-      (GitHub <a href="https://github.com/jacoco/jacoco/issues/610">#610</a>).</li>
-  <li>Exclude from a report private empty constructors that do not have arguments
-      (GitHub <a href="https://github.com/jacoco/jacoco/issues/529">#529</a>).</li>
-  <li>Exclude from a report a part of bytecode that javac generates for a
-      String in switch statement
+  <li>Additional bytecode compiled for <code>switch</code> statements on
+      <code>java.lang.String</code> values
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/596">#596</a>).</li>
-  <li>Maven aggregated reports will now also include modules of <code>runtime</code>
-      and <code>provided</code> dependencies
-      (GitHub <a href="https://github.com/jacoco/jacoco/issues/498">#498</a>,
-      <a href="https://github.com/jacoco/jacoco/issues/572">#572</a>).</li>
-  <li>JaCoCo now comes with a simple command line interface
-      (GitHub <a href="https://github.com/jacoco/jacoco/issues/525">#525</a>).</li>
-  <li>Manifests of JAR files now have <code>Automatic-Module-Name</code> for Java 9
-      (GitHub <a href="https://github.com/jacoco/jacoco/issues/565">#565</a>).</li>
 </ul>
 
 <h3>Fixed Bugs</h3>
 <ul>
   <li><code>dump</code> commands now report error when server unexpectedly
       closes connection without sending response
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/538">#538</a>).</li>
-  <li>More information about context is provided when unable to read stream during
-      analysis
-      (GitHub <a href="https://github.com/jacoco/jacoco/issues/541">#541</a>).</li>
   <li>Reduced chance of conflict with other agents
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/555">#555</a>).</li>
```

Comment:
strictly speaking this is not a fixed bug - behavior was correct and expected (PR even not labeled as such), so I'd rather say that this is a feature, WDYT?

P.S. section "Non-functional Changes" looks weird for me in general as any user-visible change is kinda functional, but this for other day/releases - maybe don't use it and rename "New Features" into "New Features and Improvements"

Reply:
I consider features as functionality I can use a product for, or the "positive use cases". Error handling I wouldn't name a feature. That not why a use a product.

Reply:
@marchof ok, with such definition this makes sense ✅